src/etos_lib/lib/debug.py

```diff
@@ -36,6 +36,11 @@ def default_secret_path(self):
         """Path to k8s secrets."""
         return os.getenv("ETOS_SECRET_PATH", "/etc/")
 
+    @property
+    def default_log_path(self):
+        """"Default log path."""
+        return os.getenv("ETOS_LOG_PATH", "/logs/log.json")
```

Comment:
Perhaps we should conform to default file system hierarchy standard, e.g /var/log for log files.

Reply:
We won't have permission to write in "/var/log" (not in /logs either) so I moved it to the home directory.